salt/ocdsdata.sls

```diff
@@ -1,12 +1,28 @@
 {% from 'lib.sls' import createuser %}
 
+# Set up the server
+
+/etc/motd:
+  file.managed:
+    - source: salt://system/ocdsdata_motd
```

Comment:
Should this file exist in the repo?

Reply:
It does. I'll add it when I'm not on a train that's about to get to Nottingham.